src/PAModel/CanvasDocument.cs

```diff
@@ -60,6 +60,7 @@ public class CanvasDocument
         internal ThemesJson _themes;
         internal ResourcesJson _resourcesJson;
         internal ParameterSchema _parameterSchema;
+        internal Dictionary<string, string> _customPageInputsMetadata;
```

Comment:
add a comment about what the content is? i.e.:
// Key is page name, value is ??